### PR TITLE
Add embedded-sht as submodule

### LINKED ISSUE #60: Pull embedded-sht as a submodule for the SVM module

### PR DESCRIPTION
**Pull request by rnestler**
Fixes #60 and #83 
Check the following:

 - [na] Breaking changes marked in commit message
 - [na] Changelog updated
 - [na] Code style cleaned (ran `make style-fix`)
 - [x] Tested on actual hardware

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,11 +6,14 @@ release_drivers=$(foreach d, $(drivers), release/$(d))
 
 all: prepare $(drivers)
 
-prepare: sgp-common/sgp_git_version.c
+prepare: sgp-common/sgp_git_version.c embedded-sht/sht-common/sht_git_version.c
```
Comment (abrauchli) [inline Makefile:9]: I would rather go with a phony `prepare-embedded-sht` target, or do we need to list that file explicitly?

Comment (rnestler) [reply Makefile:9]: Probably not. Let me try.

Comment (rnestler) [reply Makefile]: Fixed.

**Comment by rnestler** [comment]
@abrauchli Should I add  this to the changelog? IMO it is a internal implementation detail that users don't need to care about.

**Comment by abrauchli** [comment]
> @abrauchli Should I add this to the changelog? IMO it is a internal implementation detail that users don't need to care about.

no I don't think so, so I wouldn't

File: Makefile [hunk 2]
```diff
@@ -43,11 +46,11 @@ release/svm30: release/sgp30
 	export tag="$$(git describe --always --dirty)" && \
 	export pkgname="$${driver}-$${tag}" && \
 	export pkgdir="release/$${pkgname}" && \
-	(cd ../embedded-sht && make prepare) && \
+	(cd embedded-sht && make prepare) && \
```
Comment (abrauchli) [inline Makefile:49]: ist nicht einfacher dependency auf `prepare-embedded-sht` ?

File: Makefile [hunk 3]
```diff
@@ -65,11 +68,11 @@ release/sgpc3_with_shtc1: release/sgpc3
...
```
Comment (abrauchli) [inline Makefile:71]: ...same here

File: Makefile (hunk 2 above)
Comment (rnestler) [reply Makefile]: True. Fixed.

File: Makefile (hunk 3 above)
Comment (rnestler) [reply Makefile]: ...same here :wink:

**Pull request merged**